lib/commandCenter/commandCenter.js

```diff
@@ -171,7 +177,7 @@ CommandCenter.prototype.setupUserCommand = function (modName, cmdName, cmd) {
 		throw new UserCommandSetupError('Command name must be a string', logDetails);
 	}
 
-	if (!cmd || typeof cmd !== 'object') {
+	if (!cmd || (typeof cmd !== 'object' && typeof cmd !== 'function')) {
```

Comment:
Could you add a comment stating why the `function` type is valid? Mentioning classes that expose `execute` and `acl` statically.